### DISCUSSION TITLE
Sample Data for Template One / Zero?

### DISCUSSION THREAD

**Discussion by huteck** [templaza/astroid-framework#237] [Q&A] [answered]
Hi, is there no sample data available for the included templates or do I miss anything?

**Comment by Chacapamac** [comment, marked as answer]
**Not to my knowledge.**
I guess you can use the sample data that come wit Joomla **Cassiopea** and refurbished it for testing Astroid

**Info on Joomla Sample Data**
https://www.inmotionhosting.com/support/edu/joomla/joomla-4/load-sample-data/

I agree that it will be nice to have a specific Astroid Sample Data to show the great possibilities & power of the framework.

**I add the label “enhancement”**
